hist/hist/src/TF1.cxx

```diff
@@ -3168,43 +3171,44 @@ void TF1::Save(Double_t xmin, Double_t xmax, Double_t, Double_t, Double_t, Doubl
          TH1 *h = (TH1 *)fParent;
          Int_t bin1 = h->GetXaxis()->FindBin(xmin);
          Int_t bin2 = h->GetXaxis()->FindBin(xmax);
-         int fNsave = bin2 - bin1 + 4;
-         //fSave  = new Double_t[fNsave];
-         fSave.resize(fNsave);
+         int nsave = bin2 - bin1 + 4;
```

Comment:
Why are you using now `nsave` and not `fNSave` ? I don't see where `fNSave` is set to the number of saved points

Reply:
fNsave was data member some time ago. Now I rename it to avoid confusion.

Reply:
OK, thanks for clarifying this! 
It would be good to rename fSave to save also in the `TF1::GetSave` function